.travis.yml

```diff
@@ -4,4 +4,6 @@ before_script:
     - bundle install
 script:
     - xcodebuild -project Aardvark.xcodeproj -scheme Aardvark -sdk iphonesimulator -configuration Debug -PBXBuildsContinueAfterErrors=0 ACTIVE_ARCH_ONLY=0 build test
+    - xcodebuild -project Aardvark.xcodeproj -scheme "Aardvark-iOS" -sdk iphonesimulator clean
```

Comment:
do we need the `clean` prior to the run? Travis-CI should start all builds from a fresh environment.